js/modules/k6/browser/common/frame_session.go

```diff
@@ -661,7 +661,7 @@ func (fs *FrameSession) onConsoleAPICalled(event *cdpruntime.EventConsoleAPICall
 }
 
 func (fs *FrameSession) onExceptionThrown(event *cdpruntime.EventExceptionThrown) {
-	fs.page.emit(EventPageError, event.ExceptionDetails)
+	// TODO: Test and handle this
```

Comment:
I am not sure if I'm getting this todo and also, does that make sense to have this empty body method at all?

Reply:
Right, i think we need an issue for this as it may be something useful to build an API around. When the website throws an exception then the user might want to track this.

I don't want to delete it. Instead i might add some debug logging when this is called.

Reply:
I see, yeah logging that sounds reasonable, feel free to do this later, but it's worth maybe updating TODO within this PR to put more context for the future implementation :+1: